clar2wasm/src/wasm_generator.rs

```diff
@@ -1617,4 +1622,13 @@ mod misc_tests {
             evaluate("(ok false)"),
         );
     }
+
+    #[test]
+    fn end_of_standard_data_is_correct() {
+        let standard_lib_wasm: &[u8] = include_bytes!("standard/standard.wasm");
```

Comment:
Not sure I would use `include_bytes!` here. It would add all the bytes of *standard.wasm* to the test binary. Sure it's not big, but for one simple test, it seems overkill.

Reply:
Replaced with fs::read.